spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe "Mechanics Show Page" do
+  before(:each) do
+    @amusement_park = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @mechanic1 = Mechanic.create!(name: "Larry", years_experience: 3)
+    @mechanic2 = Mechanic.create!(name: "Ben", years_experience: 10)
+
+    @ride1 = @mechanic1.rides.create!(name: "The Hurler", thrill_rating: 7, open: false, amusement_park_id: @amusement_park.id)
+    @ride2 = @mechanic1.rides.create!(name: "Dragon Hunter", thrill_rating: 6, open: false, amusement_park_id: @amusement_park.id)
+    @ride3 = @mechanic2.rides.create!(name: "Twister", thrill_rating: 9, open: false, amusement_park_id: @amusement_park.id)
+    @ride4 = @mechanic2.rides.create!(name: "Merchant of Death", thrill_rating: 10, open: false, amusement_park_id: @amusement_park.id)
+    @ride5 = Ride.create!(name: "Fun Time", thrill_rating: 2, open: false, amusement_park_id: @amusement_park.id)
+  end
+
+  describe 'User Story 1' do
+    it "Displays a mechanics attributes" do
+      visit mechanic_path(@mechanic1)
+
+      expect(page).to have_content(@mechanic1.name)
+      expect(page).to have_content(@mechanic1.years_experience)
+
+      expect(page).to_not have_content(@mechanic2.name)
+      expect(page).to_not have_content(@mechanic2.years_experience)
+    end
+
+    it "Displays a mechanics rides name" do
+      visit mechanic_path(@mechanic1)
+
+      expect(page).to have_content(@ride1.name)
+      expect(page).to have_content(@ride2.name)
+
+      expect(page).to_not have_content(@ride3.name)
+      expect(page).to_not have_content(@ride4.name)
+    end
+  end
+
+  describe 'User Story 2' do
+    it "can add a ride to a mechanic" do
+      visit mechanic_path(@mechanic1)
+
+      expect(page).to have_field("Ride ID")
+
+      fill_in "Ride ID:", with: @ride5.id
+      click_button "Add Ride"
+      expect(current_path).to eq(mechanic_path(@mechanic1))
+      
+      expect(page).to have_content(@ride5.name)
+    end
+  end
```

Comment:
While this test covers the majority of the US, I think it could have benefited from first checking that the name of the ride isn't already on the page with a `to_not have_content` assertion before filling out the form.